app/static/css/styles.css

```diff
@@ -611,3 +611,16 @@ html {
       display: flex;
   }
 }
+
+/*Search Css*/
+.subjects-checkbox-container {
+    cursor: pointer;
+    max-height: 130px; /* Adjust based on your line-height and padding to show only 4 rows */
+    overflow-y: auto; /* Enable vertical scrollbar when content overflows */
+    border: 1px solid #ced4da; /* Bootstrap's form control border color */
+    border-radius: 0.25rem; /* Bootstrap's form control border radius */
+}
+
+.subjects-checkbox-container input[type="checkbox"] {
+    cursor: pointer;
+}
```

Comment:
The `cursor: pointer` above creates a few areas of strange UX: for example, just right of the label, the pointer is shown, but the label doesn't extend that far. The labels themselves activate the checkbox, but show the default (auto) cursor.

Not sure what you had in mind here, but this might need to be reconsidered.